cm.py

```diff
@@ -107,31 +107,31 @@ def main():
   profile = get_profile_for_api(config, auth, args.account)
   kwargs = { 'profileId': profile } 
 
-  # get report json
+  # get report yaml
   if args.report:
     kwargs['reportId'] = args.report
     report = API_DCM(config, auth).reports().get(**kwargs).execute()
-    print(json.dumps(report, indent=2, sort_keys=True))
+    print(yaml.dump(report, indent=2, sort_keys=True))
```

Comment:
The reporting API uses JSON, this helper should return something like:
https://developers.google.com/doubleclick-advertisers/rest/v4/reports